fog.gemspec

```diff
@@ -45,7 +45,7 @@ Gem::Specification.new do |s|
   s.extra_rdoc_files = %w[README.md]
 
   ## Ruby version
-  s.required_ruby_version = ">= 1.9.3"
+  s.required_ruby_version = ">= 2"
```

Comment:
We didn't get to a decision about dropping `1.9.3` so perhaps we need to know if for sure first.